Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.yml

```diff
@@ -19,6 +19,18 @@ configuration:
   name: base_url
   required: true
   type: 0
+- display: Use a self-deployed Azure application
```

Comment:
i think the `self_deployed` name here is confusing, because it is actually just enabled client credentials
note that this integration already supported self deployed mode for device code
lets improve display name and docs around that